README.md

```diff
@@ -82,7 +82,7 @@ Cryptography|Bitcoin, Chance and Randomness|[Bitcoin, Chance and Randomness](htt
 Cryptography|On Bitcoin Security in the Presence of Broken Crypto Primitives|[On Bitcoin Security in the Presence of Broken Crypto Primitive](http://diyhpl.us/~bryan/papers2/bitcoin/On%20Bitcoin%20security%20in%20the%20presence%20of%20broken%20crypto%20primitives%20-%202016.pdf)|
 Cryptography|Signatures and zero-knowledge proofs|[State of Cryptography - beyond ECDSA and sha256](https://scalingbitcoin.org/transcript/stanford2017/state-of-cryptography)|
 Cryptography|zero knowledge proofs|[Introduction to SNARKs](https://www.youtube.com/watch?v=jr95o_k_SwI&feature=youtu.be)|
-Cryptography|Bulletproofs|[How Bulletproofs Could Make Bitcoin Privacy Less Costly](https://bitcoinmagazine.com/articles/how-bulletproofs-could-make-bitcoin-privacy-less-costly/), [Bulletproofs: Faster Rangeproofs and Much More](https://blockstream.com/2018/02/21/bulletproofs-faster-rangeproofs-and-much-more/), [Bulletpoints on bulletproofs](https://joinmarket.me/blog/blog/bulletpoints-on-bulletproofs/), [From Zero Knowledge Proofs to Bulletproofs Paper](https://joinmarket.me/blog/blog/from-zero-knowledge-proofs-to-bulletproofs-paper/), [Building on Bulletproofs](https://medium.com/@cathieyun/building-on-bulletproofs-2faa58af0ba8)|
+Cryptography|Bulletproofs|[How Bulletproofs Could Make Bitcoin Privacy Less Costly](https://bitcoinmagazine.com/articles/how-bulletproofs-could-make-bitcoin-privacy-less-costly/), [Bulletproofs: Faster Rangeproofs and Much More](https://blockstream.com/2018/02/21/bulletproofs-faster-rangeproofs-and-much-more/),  [Building on Bulletproofs](https://medium.com/@cathieyun/building-on-bulletproofs-2faa58af0ba8)|
```

Comment:
such a shame that waxwing's blog is down. maybe substitute https://medium.com/@nopara73/confidential-transactions-bulletproofs-the-elephant-in-the-room-cfdb37ce509?